multi_import/helpers/files.py

```diff
@@ -15,14 +15,27 @@ def find_format(file_formats, file_format=None):
     )
 
 
+def decode_contents(file_contents):
```

Comment:
Taking a step back, can't we just rely on the `chardet` library to detect the file encoding we should use? And by using the `chardet` library I think we can drop the encoding specific tests. We are already using chardet in for the CSV format to detect the file encoding to ensure it is unicode

Reply:
Thanks for the comment @RouganStriker! We tried `chardet.detect()` before and it somehow gave `None` for encoding value for certain files (ex: `howto.xlsx` exported from our app) and an error occured since `file_contents.decode(None)` is not valid :'( Then, we tried few options, and 'ISO-8859-1(Latin-1)' worked..
```
(Pdb) chardet.detect(file_contents)
{'encoding': None, 'confidence': 0.0, 'language': None}
```